js/models/TimeTrigger.js

```diff
@@ -4,8 +4,8 @@ TimeTrigger.schema = {
   primaryKey: 'id',
   properties: {
   	id: 'string',
-  	formId: 'string',
-  	surveyId: 'string',
+  	formId:  {type: 'string', default: 'none'},
+  	surveyId:  {type: 'string', default: 'none'},
```

Comment:
Are we checking for 'none' somewhere that we are defaulting to this value? Perhaps a preference thing, but why not null?